### ISSUE TITLE
Export `WebVRPolyfill` as standalone module

### ISSUE THREAD

**Issue by cvan** [googlevr/webvr-polyfill#47]
In [`main.js`](https://github.com/borismus/webvr-polyfill/blob/master/src/main.js), instead of calling `new WebVRPolyfill();` or `window.InitializeWebVRPolyfill` (which calls `new WebVRPolyfill();` at some later time), `window.WebVRPolyfill` should be exposed for browsers and for CommonJS `module.exports = WebVRPolyfill`.

This allows folks to use `window.WebVRPolyfill` or `require('webvr-polyfill')` in browsers and CommonJS, respectively.

And browserify will handle the conditional UMD (window, CommonJS, AMD) exporting for you. Just need to change `src/main.js` and have browserify export a standalone bundle.

**Comment by cvan** [comment]
This isn't fixed. Why is this closed?

**Comment by borismus** [comment]
I thought I had fixed it, but you're right!

**Comment by jsantell** [comment]
Currently you can access the `WebVRPolyfill` object in built environments (browserify, webpack, etc.) via 

```js
// es6 import
import { WebVRPolyfill } from 'webvr-polyfill/src/webvr-polyfill';
// require
const WebVRPolyfill = require('webvr-polyfill/src/webvr-polyfill').WebVRPolyfill;
```

The use case of dropping in the script file should just seemlessly polyfill the API (which is what bundling from src/main.js does), and any custom configuration (beyond the WebVRConfig values) where you'd want to access the polyfill constructor directly can be done by accessing the constructor via require/es6 imports. Is there a common use case inbetween these two scenarios?

**Comment by cvan** [comment]
I started a local branch on this. Is there interest in it? (I've been modelling it after `es6-promise`.)

**Comment by jsantell** [comment]
I'd just be hesitant to do any big architectural changes without first having some solid testing, and that the script in `build/` still just auto-polyfills without further interaction, and if the imports work from https://github.com/googlevr/webvr-polyfill/issues/47#issuecomment-303794167, I'd be curious what is missing (in lieu of larger changes)

**Comment by cvan** [comment]
à la [`es6-promise`](https://github.com/stefanpenner/es6-promise#auto-polyfill), this is what I'd recommend:

```js
// es6 import
import { WebVRPolyfill } from 'webvr-polyfill';
WebVRPolyfill.polyfill();

// require
const WebVRPolyfill = require('webvr-polyfill');
WebVRPolyfill.polyfill();
```

**Comment by jsantell** [comment]
Fixed in [0.10.0](https://github.com/googlevr/webvr-polyfill/releases/tag/v0.10.0)
